src/seedu/addressbook/commands/Command.java

```diff
@@ -39,9 +40,7 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public CommandResult execute(){
-        throw new UnsupportedOperationException("This method should be implement in child classes");
-    }
+    public abstract CommandResult execute();
 
```

Comment:
I believe this is for LO6.1f? Try to make a clean PR in the future. You can figure out how to stage only selected lines instead of the entire file before committing.